src/service/vsync-impl.js

```diff
@@ -410,11 +421,25 @@ export class Vsync {
         }
       }
     }
+
+    if (getMode().localDev && !getMode().test &&
+        this.ampdocService_.isSingleDoc()) {
```

Comment:
`if (getMode().localDev && this.mutationPolice_)` to simplify while preserving DCE.